TLM/TLM/Patch/_VehicleAI/_AmbulanceAI/StartPathFindPatch.cs

```diff
@@ -0,0 +1,25 @@
+namespace TrafficManager.Patch._VehicleAI._AmbulanceAI {
+    using HarmonyLib;
+    using JetBrains.Annotations;
+    using TrafficManager.Patch._PathManager;
+    using System.Reflection;
+    using TrafficManager.API.Traffic.Enums;
+    using TrafficManager.Manager.Impl;
+    using ColossalFramework;
+
+    [HarmonyPatch]
+    public class StartPathFindPatch {
+        [UsedImplicitly]
+        public static MethodBase TargetMethod() => StartPathFindCommons.TargetMethod<AmbulanceAI>();
+
+        [UsedImplicitly]
+        public static void Prefix(ushort vehicleID, ref Vehicle vehicleData) {
```

Comment:
@kianzarrin did that work as intended at a time of merging?
Original method signature was `ushort vehicleId, ref Vehicle vehicleData, Vector3 startPos, Vector3 endPos, bool startBothWays, bool endBothWays, bool undergroundTarget` and in fact, we are searching for the method with such signature (using delegate description of signature) but prefixing it only with `ushort vehicleID, ref Vehicle vehicleData`.
It's similar for other `StartPathFind` prefixes. 
Am I missing something here?